addons/web/static/src/views/fields/image/image_field.js

```diff
@@ -47,7 +47,11 @@ export class ImageField extends Component {
         onWillUpdateProps((nextProps) => {
             const { record } = this.props;
             const { record: nextRecord } = nextProps;
-            if (record.resId !== nextRecord.resId || nextRecord.mode === "readonly") {
+            if (
+                (!this.rawCacheKey && !isBinarySize(this.props.value)) ||
```

Comment:
`isBinarySize(nextProps.value))` I think

and a comment that says in substance that if it is a binarySize, it means that the image is a url ; otherwise it is a base64 value

Reply:
Hmm, I think you need to pause the review, it doesn't work when changing any other field.